packages/components/src/custom-select-control-v2/legacy-component/test/index.tsx

```diff
@@ -286,7 +297,7 @@ describe.each( [
 			expect.objectContaining( {
 				inputValue: '',
 				isOpen: false,
-				selectedItem: { key: 'violets', name: 'violets' },
+				selectedItem: { key: 'flower1', name: 'violets' },
```

Comment:
Need to double-check if this behaviour matches v1

Reply:
v1 doesn't behave this way, because it doesn't fire `onChange` on first render. This is something that we should investigate separately from this PR, though